configuration-files/README.md

```diff
@@ -135,7 +134,7 @@ curl -i http://localhost:8080/api/v1/service-a
 # X-Service: service-a
 ```
 
-The `SERVICE_NAME` environment value `example` got also replaced with `service-a`. 
+The `SERVICE_NAME` environment value `example` got also replaced with `service-a`.
 
 You may have read how the files will be merged by file-name-order. A `couper.hcl` will be prioritized within this directory
 which makes this file a good starting point for our base configuration.
```

Comment:
Drop last sentence or whole paragraph.

Reply:
dropped, load hint is mentioned at the beginning.